test/metabase/query_processor_test/string_extracts_test.clj

```diff
@@ -55,19 +53,15 @@
     (is (= "ne" (test-string-extract [:substring [:field (data/id :venues :name) nil]
                                       [:- [:length [:field (data/id :venues :name) nil]] 1]])))))
 
+
 (deftest ^:parallel test-replace
   (mt/test-drivers (mt/normal-drivers-with-feature :expressions)
-    (when (or (not= driver/*driver* :mongo)
```

Comment:
We only support/test against Mongo 4.4+ these days so I was able to remove this version check